src/main/ts/components/Editor.ts

```diff
@@ -56,7 +57,7 @@ export const Editor = defineComponent({
         inline: inlineEditor,
         setup: (editor: any) => {
           vueEditor = editor;
-          editor.on('init', (e: Event) => initEditor(e, props, ctx, editor, modelValue, getContent()));
+          editor.on('init', (e: Event) => initEditor(e, props, ctx, editor, modelValue, content));
```

Comment:
Could the content/model value have changed by the time it's being used here?

Reply:
Changed to a fn for v-model to get the latest content